README.md

```diff
@@ -26,42 +26,110 @@ module "s3" {
+data "aws_iam_policy_document" "s3_migrate_policy" {
+  # List & location for source & destination S3 bucket.
+  statement {
+    actions = [ 
+      "s3:ListBucket",
+      "s3:GetBucketLocation"
+    ]
+    resources = [ 
+      module.s3_bucket.bucket_arn,
+      "arn:aws:s3:::[source-bucket-name]"
+    ]
+  }
+  # Permissions on source S3 bucket contents. 
+  statement {
+    actions = [ 
+      "s3:GetObject",
+      "s3:GetObjectVersion",
+      "s3:GetObjectTagging"
+    ]
+    resources = [ "arn:aws:s3:::[source-bucket-name]/*" ]   # take note of trailing /* here
+  }
+  # Permissions on destination S3 bucket contents. 
+  statement {
+    actions = [
+      "s3:PutObject",
+      "s3:PutObjectTagging",
+      "s3:GetObject",
+      "s3:DeleteObject"
+    ]
+    resources = [ "${module.s3_bucket.bucket_arn}/*" ]
+  }
+}
+
+resource "aws_iam_policy" "s3_migrate_policy" {
+  name   = "s3_migrate_policy"
+  policy = data.aws_iam_policy_document.s3_migrate_policy.json
+
+  tags = {
+    business-unit          = var.business_unit
+    application            = var.application
+    is-production          = var.is_production
+    environment-name       = var.environment
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure_support
+  }
+}
 
-Example to retrieve destination IAM user for use in source bucket policy. _requires [jq - commandline JSON processer](https://stedolan.github.io/jq/)_
+# store irsa rolearn in k8s secret for retrieving to provide within source bucket policy
+resource "kubernetes_secret" "cross_irsa" {
+  metadata {
+    name      = "cross-irsa-output"
+    namespace = var.namespace
+  }
+  data = {
+    role           = module.cross_irsa.role_name
+    rolearn        = module.cross_irsa.role_arn
+    serviceaccount = module.cross_irsa.service_account.name
+  }
+}
 
-```bash
-# retrieve destination s3 user ARN
+# set up the service pod
+module "cross_service_pod" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-service-pod?ref=[latest-release-here]"
+  namespace            = var.namespace
+  service_account_name = module.cross_irsa.service_account.name
+}
+```
 
-# retrieve live-1 namespace's s3 credentials
-$ kubectl -n my-namespace get secret my-s3-secrets -o json | jq -r '.data[] | @base64d'
-=>
-<access_key_id>
-<bucket_arn>
-<bucket_name>
-<secret_access_key>
 
-# retrieve IAM user details using credentials
-$ unset AWS_PROFILE; AWS_ACCESS_KEY_ID=<access_key_id> AWS_SECRET_ACCESS_KEY=<secret_access_key> aws sts get-caller-identity
+#### Source bucket policy
+
+The source bucket must permit your IRSA role to "read" from its bucket explcitly.
 
-# Alternative single call in bash
-$ unset AWS_PROFILE; read K a n S <<<$(kubectl -n my-namespace get secret my-s3-secrets -o json | jq -r '.data[] | @base64d') ; AWS_ACCESS_KEY_ID=$K AWS_SECRET_ACCESS_KEY=$S aws sts get-caller-identity
+# retrieve IRSA rolearn using cloud-platform CLI and [jq](https://jqlang.github.io/jq/)
```

Comment:
This looks a heading when u view them in the UI. probably should be under the below code block